modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/config/DynamicConfigWithStatefulOperationConfigurationOverrideTestCase.java

```diff
@@ -6,23 +6,41 @@
  */
 package org.mule.test.module.extension.config;
 
-import org.mule.test.module.extension.InvalidExtensionConfigTestCase;
+import org.mule.runtime.api.lifecycle.InitialisationException;
+import org.mule.test.module.extension.AbstractExtensionFunctionalTestCase;
 
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
-public class DynamicConfigWithStatefulOperationConfigurationOverrideTestCase extends InvalidExtensionConfigTestCase {
+// TODO MULE-19352 migrate this test to InvalidExtensionConfigTestCase
```

Comment:
use an appropriate jdoc and allure and add the TODO in there.

Reply:
this is just a TODO, not a description of the test